index.js

```diff
@@ -68,7 +91,21 @@ function checkIfAnyMovieHasRating() {}
       // Toy Story 4
     };
  */
-function findById() {}
+const findById = (movies, id)=> {
+  let result = {}
+  
+  if (!movies.length){
+    throw " there are no movies"
+  }
+  
+  let findArr = movies.find((movie) => {
+    if(movie.imdbID === id){
+      return movie.title
+    } 
+    
+  })
+  return findArr
```

Comment:
If .find cannot find a matching movie, it will return undefined. You want to return "null" instead of "undefined", so make a conditional to account for that.
```suggestion
  if(findArr === undefined) {
  return null
  } else {
  return findArr
  }
```